Mogakco/Sources/Data/DataSources/Local/UserDefaultsUserDataSource.swift

```diff
@@ -11,8 +11,10 @@ import Foundation
 import RxSwift
 
 struct UserDefaultsUserDataSource: LocalUserDataSourceProtocol {
+    
     private let userKey = "user"
-
+    private let userUIDKey = "userUID"
```

Comment:
```suggestion
    private let userIDKey = "userID"
```
대부분 ID로 변수명이 통일되어 있어서 ID로 변경하면 좋을 것 같아요!